examples/logging.pp

```diff
@@ -0,0 +1,51 @@
+# Sensu Go docs: https://docs.sensu.io/sensu-go/latest/guides/systemd-logs/
```

Comment:
This needs links to the rsyslog and logrotate modules. Perhaps a `Puppetfile_logging` or similar and a comment to run a command to install them from that file.